app/src/main/AndroidManifest.xml

```diff
@@ -11,16 +12,21 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".activity.MainActivity">
+
+        <!-- Activities -->
+        <activity
+            android:name=".ui.main.MainActivity"
+            android:label="@string/app_name"
+            android:theme="@style/AppTheme.NoActionBar">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".ui.event.EventsActivity"></activity>
+        <activity android:name=".ui.events.EventsActivity"></activity>
```

Comment:
Jeg synes den er mer oversiktlig. Akkurat nå lå jo alt sammen bare i samme mappe. Ganske utbredt måte å organisere på etter hva jeg har sett, tanken er at man organiserer UI komponenter etter features/roller. 
Se: 
- https://overflow.buffer.com/2016/09/26/android-rethinking-package-structure/
- http://stackoverflow.com/questions/37664828/android-mvp-pattern-package-structure
- https://medium.com/@kailash09dabhi/mvp-with-better-naming-of-implementation-classes-dry-principle-e8b6130bbd02#.x444z2tog